app/src/pages/inside/common/filterEntitiesGroups/suiteLevelEntities.jsx

```diff
@@ -150,6 +151,7 @@ const DEFECT_ENTITY_ID_BASE = 'statistics$defects$';
   projectId: activeProjectSelector(state),
```

Comment:
Here you decided to leave `projectId`? Why? In comparison with other entity types.